docs/sources/administration/api-keys/index.md

```diff
@@ -45,30 +53,95 @@ This topic shows you how to create an API key using the Grafana UI. You can also
-1. Click **Administration** in the left-side menu and select **API Keys**.
-1. In the top of the page, find the section that says Switch from API keys to service accounts.
+### Migrate API keys to Grafana service accounts using the Grafana user interface
+
+This section shows you how to migrate API keys to Grafana service accounts using the Grafana user interface. You can choose to migrate a single API key or all API keys. When you migrate all API keys, you can no longer create API keys and must use service accounts instead.
+
+#### Before you begin
+
+To follow these instructions, you need one of the following:
+
+- Administrator permissions
+- Editor permissions
+- Service account writer
+
+For more information about permissions, refer to [Roles and permissions]({{< relref "../roles-and-permissions/#" >}}).
+
+#### Steps
+
+To migrate all API keys to service accounts, complete the following steps:
+
+1. Sign in to Grafana, hover your cursor over **Configuration** (the gear icon), and click **API Keys**.
+1. In the top of the page, find the section which says **Switch from API keys to service accounts**
 1. Click **Migrate to service accounts now**.
 1. A confirmation window will appear, asking to confirm the migration. Click **Yes, migrate now** if you are willing to continue.
 1. Once migration is successful, you can choose to forever hide the API keys page. Click **Hide API keys page forever** if you want to do that.
 
-**To migrate single API key to a service account:**
+To migrate a single API key to a service account, complete the following steps:
 
 1. Sign in to Grafana.
 1. Click **Administration** in the left-side menu and select **API Keys**.
 1. Find the API Key you want to migrate.
 1. Click **Migrate to service account**.
+
+### Migrate API keys to Grafana service accounts using the API
+
+This section shows you how to migrate API keys to Grafana service accounts using the Grafana API.
+
+#### Before you begin
+
+To follow these instructions, you need one of the following:
+
+- Administrator permissions
+- Editor permissions
+- Service account writer
+
+#### Steps
+
+Complete the following steps to migrate from API keys to service accounts using the API:
+
+1. Call the `POST /api/serviceaccounts` endpoint and the `POST /api/serviceaccounts/<id>/tokens`.
+
+   This action generates a service account token.
+
+1. Store the ID and secret that the system returns to you.
+1. Pass the token in the `Authrorization` header, prefixed with `Bearer`.
+
+   This action authenticates API requests.
+
+1. SATs used for authentication
+1. Remove code that handles the old `/api/auth/keys` endpoint.
+1. Track the [API keys](http://localhost:3000/org/apikeys) in use and migrate them to SATs.
+
+### Migrate API keys to Grafana service accounts using Terraform
+
+This section shows you how to migrate API keys to Grafana service accounts using Terraform.
+
+#### Before you begin
+
+To follow these instructions, you need:
+
+- N/A
```

Comment:
Is this truly Not Applicable? Or is it a placeholder for info to be added later?

Reply:
as long as you have access to your terraform configuration, anyone that has that (regardless of the permissions in Grafana)